openmc/data/njoy.py

```diff
@@ -50,84 +50,66 @@
     75: ThermalTuple('ouo2', [8016, 8017, 8018], 1),
 }
 
-
-_PENDF_TEMPLATE = """
+_ACE_TEMPLATE_RECONR = """
```

Comment:
We might as well rename all these to just `_TEMPLATE_*` since they are not specific to ACE now.